lib/beaker-hostgenerator/data.rb

```diff
@@ -391,6 +391,48 @@ def osinfo
                           'template' => 'redhat-9-arm64',
                         },
                       },
+                      'rocky8-64' => {
```

Comment:
I'm not a big fan of this repetition, but I currently don't see a better way to implement this.